CRM/Banking/PluginImpl/Exporter/CSV.php

```diff
@@ -273,7 +273,15 @@ protected function apply_rule($rule, &$data_blob) {
           $data_blob[$rule->to . $key] = $value;
         }
       }
-
+    } elseif ($rule->type == 'append') {
```

Comment:
`elseif` should be on a spearate line.
Actually, the entire `if`-`elseif`-`else` chain for `$rule->type` could be a `switch`-`case`-`default` for better readability.